prime-router/src/main/kotlin/transport/RedoxTransport.kt

```diff
@@ -56,10 +56,8 @@ class RedoxTransport() : ITransport, SecretManagement {
             // DevNote: Redox access tokens live for many days
             val token = fetchToken(redoxTransportType, key, secret, context)
             if (token == null) {
-                actionHistory.trackActionResult(
-                    "Failure: fetch redox token failed.  Requesting retry of allItems"
-                )
-                return RetryToken.allItems // finally block below will execute to record history
```

Comment:
This line 62 was the bug.

Reply:
Right. Do not turn retryItems into allItems.